sunpy/net/dataretriever/tests/test_rhessi.py

```diff
@@ -82,8 +82,8 @@ def test_get_base_url_on_timeout(mock_urlopen):
 
 def parsed_dbase():
     """
-    The result of calling `parse_observing_summary_dbase_file(...)` on
-    https://hesperia.gsfc.nasa.gov/hessidata/dbase/hsi_obssumm_filedb_200311.txt but
+    The result of calling `parse_observing_summary_dbase_file(...)` on https://
+    hesperia.gsfc.nasa.gov/hessidata/dbase/hsi_obssumm_filedb_200311.txt but
     only using the first two rows of data.
```

Comment:
```suggestion
    The result of calling `parse_observing_summary_dbase_file(...)` on
    https://hesperia.gsfc.nasa.gov/hessidata/dbase/hsi_obssumm_filedb_200311.txt
    but only using the first two rows of data.
```
Not terribly important since this is in a test, but I don't think URLs should be broken unless they really really need to be.

Reply:
Yeah it has issues with URLs. I think its worth doing on a one off basis for now but not to add it to the pre-commit/CI.